add cw-receiver-minimal

LeTurt333 opened the pull request:
PR to add a simple example of implementing the cw20 receiver interface (with some added notes) because someone asked about this in Discord the other day

ueco-jb reviewed (changes requested):
Alright, I stopped the review.
If this contract supposed to show how to implement `Cw20Receive` interface, there are already existing examples:
- `cw4-stake` https://github.com/CosmWasm/cw-plus/blob/24af1b9bff0fb4dcd5592fdd546dd56d135ad52a/contracts/cw4-stake/src/msg.rs#L39-L40
- `cw20-ics20` https://github.com/CosmWasm/cw-plus/blob/24af1b9bff0fb4dcd5592fdd546dd56d135ad52a/contracts/cw20-ics20/src/msg.rs#L33-L34
- and sending example in `cw20-base` https://github.com/CosmWasm/cw-plus/blob/bc339368b1ee33c97c55a19d4cff983c7708ce36/contracts/cw20-base/src/contract.rs#L386-L392 (it implements `ExecuteMsg` from `cw20` crate)

not to mention that in this repository I think all `cw20`-related contracts implements that receive interface already as well
https://github.com/CosmWasm/cw-tokens/blob/23f55c626d17c57124e3766a4d4e3dcf2392085f/contracts/cw20-atomic-swap/src/msg.rs#L23-L24

LeTurt333 commented:
@ueco-jb Ok I fixed the mentioned issues and removed some extra files to make it align more with the other contracts 

When I saw that question asked I thought it would be a good idea to have something that shows the absolute bare minimum of what the receiver interface looks like, but if this is an unnecessary addition I totally understand
